Tasks/AzureDevOpsTestPlanV0/automatedTests.ts

```diff
@@ -0,0 +1,58 @@
+import tl = require('azure-pipelines-task-lib/task');
+import { testInvoker } from './automatedTestInvoker'
+import { TestPlanData, getAutomatedTestData } from './getAutomatedTests';
+import { TestCase } from 'azure-devops-node-api/interfaces/TestPlanInterfaces';
+
+
+export async function automatedTestsFlow() {
+
+    let ListOfTestsToBeExecuted: string[] = [];
+
+    console.log(tl.loc('automatedTestsTriggered'));
+    await getFQNsOfAutomatedTestCases()
+        .then((testsToBeExecuted) => {
+            ListOfTestsToBeExecuted = testsToBeExecuted;
+        })
+        .catch((error) => {
+            console.error("Promise rejected:", error);
+        });
+
+    await sleep(2000);
```

Comment:
For my understanding: Why is this sleep required?

Reply:
removed it, not required